bezier-rs/lib/src/lib.rs

```diff
@@ -756,6 +810,10 @@ mod tests {
 		utils::dvec2_compare(p1, p2, MAX_ABSOLUTE_DIFFERENCE).all()
 	}
 
+	fn compare_vec_of_points(vec1: Vec<DVec2>, vec2: Vec<DVec2>, max_absolute_difference: f64) -> bool {
```

Comment:
Its not really clear from the function signature what this function is doing. Add a comment or adjust the name